website/docs/developer/decentralized-renderer.md

```diff
@@ -0,0 +1,11 @@
+---
+id: decentralized-renderer
+title: Decentralized Renderer
+sidebar_label: Decentralized Renderer
+---
+
+Overview
+
+## Header
+
+TBD
```

Comment:
there are example template renderers if users want to kick start their own, comes in 3 flavours:

- https://github.com/Open-Attestation/decentralized-renderer-react-template
- https://github.com/Open-Attestation/decentralized-renderer-svelte-template
- https://github.com/Open-Attestation/decentralized-renderer-vue-template

maybe can write 1 or 2 liner to point them to the respective repos